packages/core/minos-microservice-networks/minos/networks/discovery/connectors.py

```diff
@@ -104,7 +101,8 @@ def _endpoints_from_config(config: Config) -> list[dict[str, Any]]:
         for name in config.get_services():
             decorators = EnrouteCollector(name, config).get_rest_command_query()
             endpoints += [
-                {"url": decorator.url, "method": decorator.method} for decorator in set(chain(*decorators.values()))
+                {"url": decorator.url, "method": decorator.method} | decorator.kwargs
+                for decorator in set(chain(*decorators.values()))
```

Comment:
Could you add a test to check this code?

Reply:
Done!